Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/list/VaccinationListFragment.kt

```diff
@@ -82,6 +96,12 @@ class VaccinationListFragment : Fragment(R.layout.fragment_vaccination_list), Au
                             navigatorExtras
                         )
                     }
+                    is DeleteVaccinationEvent -> {
+                        showDeleteVaccinationDialog(event.vaccinationCertificateId, event.position)
+                    }
+                    is NavigateBack -> {
+                        popBackStack()
+                    }
```

Comment:
Just a personal taste :)  , no need for `{}`  if it just one statement
```kotlin
 is DeleteVaccinationEvent -> showDeleteVaccinationDialog(event.vaccinationCertificateId, event.position)
 is NavigateBack ->  popBackStack()
```

Reply:
Adapted.